packages/k/kolourpaint/pspec_x86_64.xml

```diff
@@ -196,6 +196,11 @@
             <Path fileType="localedata">/usr/share/locale/zh_HK/LC_MESSAGES/kolourpaint.mo</Path>
             <Path fileType="localedata">/usr/share/locale/zh_TW/LC_MESSAGES/kolourpaint.mo</Path>
             <Path fileType="data">/usr/share/metainfo/org.kde.kolourpaint.appdata.xml</Path>
+            <Path fileType="data">/usr/share/xdg/colors/40.colors</Path>
+            <Path fileType="data">/usr/share/xdg/colors/Oxygen.colors</Path>
+            <Path fileType="data">/usr/share/xdg/colors/Rainbow.colors</Path>
+            <Path fileType="data">/usr/share/xdg/colors/Royal.colors</Path>
+            <Path fileType="data">/usr/share/xdg/colors/Web.colors</Path>
```

Comment:
These files conflict with kdelibs4support. You want to backport https://invent.kde.org/graphics/kolourpaint/-/merge_requests/79